src/tests/example.spec.js

```diff
@@ -158,4 +158,79 @@ test.describe('Saucedemo app basic tests', () => {
             expect(cartPrice).toBe(selectedProducts[i].price);
         }
     });
+
+    test('check info about several added random products and Total Price on the "Checkout: Overview" page', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        // get all the elements of the products
+        const products = await app.inventory.product;
```

Comment:
Also, here. It's better to have this like `const products = await app.inventory.products;` instead of `app.inventory.product;`
It is better to use plural form here like u did there:
`const checkoutItems = await app.checkoutStepTwo.cartItems;`
Or u can pick whatever form of the word u like and use it everywhere. It's something called codestyle. Having different forms looks like mess